website/docs/src/content/cli-reference/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: CLI Reference
```

Comment:
Let's call this "Overview" or "CLI Overview" or something like that. Right now "CLI Reference" is duplicated and it makes it look bad 
![image](https://github.com/user-attachments/assets/2abdf308-c058-44df-b828-6e0d93820311)

```suggestion
title: Overview
```